examples/nanoChatGPT/data/shakespeare.py

```diff
@@ -49,6 +47,19 @@ def __getitem__(self, idx):
             batch_size=[self.block_size],
         )
 
+    def __getitems__(self, idx):
```

Comment:
the collector will look for getitems. If found, it will use that instead. With this implementation, we don't need to stack.